Storage/src/commonMain/kotlin/io/github/jan/supabase/storage/BucketApiImpl.kt

```diff
@@ -203,32 +215,44 @@ internal class BucketApiImpl(override val bucketId: String, val storage: Storage
     override suspend fun list(
         prefix: String,
         filter: BucketListFilter.() -> Unit
-    ): List<BucketItem> {
+    ): List<FileObject> {
         return storage.api.postJson("object/list/$bucketId", buildJsonObject {
             put("prefix", prefix)
             putJsonObject(BucketListFilter().apply(filter).build())
         }).safeBody()
     }
 
+    override suspend fun info(path: String): FileObjectV2 {
+        val response = storage.api.get("object/info/$bucketId/$path")
```

Comment:
Hey, lets wait a bit for adding the `info` method as there is a bug in the JS lib, this should be `object/info/public`, but there is also a `object/info/authenticated`.

We're figuring out internally on how we're naming methods.

So lets just wait a bit before merging this.

Reply:
Yea, I [commented](https://github.com/supabase/storage-js/pull/207#issuecomment-2301793713) on the JS PR regarding this. I couldn't make this work on the hosted Supabase instance, but the self-hosted Docker one works.

Reply:
@grdsdev Any news regarding this?